econml/tests/test_policy_forest.py

```diff
@@ -290,6 +290,8 @@ def test_non_standard_input(self,):
         forest = PolicyForest(n_estimators=20, n_jobs=1, random_state=123).fit(X, y)
         pred = forest.predict(X)
         pred_val = forest.predict_value(X)
+        pred_prob = forest.predict_proba(X)
+        np.testing.assert_allclose(pred_prob.shape, (X.shape[0], 2))

```

Comment:
Shapes should be equal, not merely close :)
```suggestion
        assert pred_prob.shape == (X.shape[0], 2)
```